python/amico/core.py

```diff
@@ -51,8 +52,9 @@ def __init__( self, study_path, subject ) :
 
         self.set_config('peaks_filename', None)
         self.set_config('doNormalizeSignal', True)
+        self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
-
+        self.set_config('doSaveCorrectedDWI', False)
```

Comment:
you are right. this should not be there... let me think about it. We can sit down together and fix this.